docs/app/devbox.json

```diff
@@ -1,12 +1,31 @@
 {
   "packages": [
+    "asciinema",
+    "vim",
+    "emacs",
+    "direnv",
+    "yq",
+    "watch",
+    "openssl",
+    "dive",
+    "tree",
+    "tmux",
     "yarn",
-    "nodejs"
+    "httpie",
+    "git",
+    "ripgrep",
+    "glow",
+    "gifsicle",
+    "epr",
+    "nixfmt",
+    "pulumi-bin",
+    "rnix-lsp",
+    "graphviz"
```

Comment:
why all these packages?

Reply:
That's weird, those are my global packages. They shouldn't be included

Reply:
Fixed